### ISSUE TITLE
IV tracker export tweaks

### ISSUE THREAD

**Issue by kuoxes1** [seemack/healthy-hearts-northwest-dev#92]
the coach managers would like:
we are wondering if:
- we could see the meeting attendees (could this come up as a single text box w/names & roles?),
-we could see the coaches' PDSA info., and
-we could receive weekly data pulls with just the newly added entries (NOT AS MUCH OF A PRIORITY

**Comment by kuoxes1** [comment]
1. Can we see list of meeting attendees: going to talk to redcap people and Caitlin to see.  don't want to slow down the export too much
 
2. see the coach PDSA info: again talk tot hem - this might be something we ask them to do in Redcap

3. Make sure IV tracker sorts by date.

Clarify what they really want:  Leah is going to hook Chris up with their redcap people

**Comment by seemack** [comment]
Caitlyn & I talked.  I suggested she mock up an Excel spreadsheet to ensure that it'll work in her REDCap instance.  She thinks what will work best is adding fields to the current IVTracker export:

- a single column with all meeting attendees & their roles
- 10 additional columns for the (up to 10) PDSAs associated with each contact
- (the sorting by date is easy - I can do that in any case) 

I explained there's a possibility this will slow down the system to an unacceptable point, and if that's the case - or if after fiddling with their own Excel ideas, they realize this won't work - Caitlyn & I will be in touch.

**Comment by kuoxes1** [comment]
Chris will reach out to Caitlin and let her know this is lower priority but we'll get to it in mid July.

**Comment by kuoxes1** [comment]
For a final analytic dataset, we do want to have something like this but probably not earlier in an excel for coach managers?

**Comment by kuoxes1** [comment]
1.  Do we need this real time or just at the end of the study? (ask coach managers)
2.  Do we need actual people or just the role? (ask coach managers)

3.  We know MICHAEL needs role for analytic purposes at some point, but not until end of study
Note:  iv contacts export is already super slow

**Comment by seemack** [comment]
Coaches did not ask for this in follow up.
